docs: update general graphcast docs, add a page for 3la radio (#63)
graphops/docs: docs/graphcast/radios/3la-monitor.md

| `LOG_FORMAT`               | Options: `pretty` - verbose and human readable; `json` - not verbose and parsable; `compact` - not verbose and not parsable; `full` - verbose and not parsible. Default value: `pretty`.                                           |
| `PERSISTENCE_FILE_PATH`    | Relative path. If set, the Radio will periodically store states of the program to the file in json format (off by default).                                                                                                        |
| `DISCV5_ENRS`              | Comma separated ENRs for Waku Discv5 bootstrapping. Defaults to empty list.                                                                                                                                                        |
| `DISCV5_PORT`              | Discoverable UDP port. Default: `9000`                                                                                                                                                                                             |

> | `ID_VALIDATION`            | Defines the level of validation for message signers used during radio operation. Options include: `no-check`, `valid-address`, `graphcast-registered`, `graph-network-account`, `registered-indexer`, `indexer` |

Reviewer: ```suggestion
| `ID_VALIDATION`            | Defines the level of validation for message signers used during Radio operation. Options include: `no-check`, `valid-address`, `graphcast-registered`, `graph-network-account`, `registered-indexer`, `indexer` |
```

### Configurations explained

#### COVERAGE (topic)
`COVERAGE` is used to specify the topic coverage level. It controls the range of topics (subgraph ipfs hashes) the Indexer subscribes to in order to process data and participate in the network.

There are three coverage levels available:

- **comprehensive**: Subscribe to on-chain topics, user-defined static topics, and subgraph deployments syncing on graph node. This level is useful for Indexers who want to compare NPOIs for all deployments syncing on their graph node even if they don't have an active allocations open (their stake will not be taken into account in attestation).
- **on-chain**: Subscribe to on-chain topics and user-defined static topics. This is the default coverage level and is suitable for indexers who only want to compare data for deployments with active allocations.
- **minimal**: Only subscribe to user-defined static topics. This level is for Indexers who want to limit their participation to specific topics of interest.

#### Identity validaiton

> `ID_VALIDATION` is used to define level of validation for message signers used during radio operation.

Reviewer: ```suggestion
`ID_VALIDATION` is used to define level of validation for message signers used during Radio operation.
```

Available Options:

> - **no-check**: No check on the message signature and do not verify the signer.

Reviewer: ```suggestion
- **no-check**:  Doesn't perform check on the message signature and does not verify the signer.
```

> - **valid-address**: signer needs to be a valid Eth address.

Reviewer: ```suggestion
- **valid-address**: Requires the signer to be a valid Ethereum address.
```

> - **graphcast-registered**: signer must be registered at Graphcast Registry.

Reviewer: ```suggestion
- **graphcast-registered**: Requires the signer to be registered on the Graphcast Registry.
```

> - **graph-network-account**: signer must be a Graph account.

Reviewer: ```suggestion
- **graph-network-account**: Signer must be a Graph account.
```

> - **registered-indexer**: signer must be registered at Graphcast Registry and correspond to an Indexer satisfying the indexer minimum stake requirement.

Reviewer: ```suggestion
- **registered-indexer**:  Requires the signer to be registered on the Graphcast Registry and corresponds to an Indexer that satisfies the minimum stake requirement.
```

> - **indexer**: signer must be registered at Graphcast Registry or is a Graph Account, and correspond to an Indexer satisfying the indexer minimum stake requirement.

Reviewer: ```suggestion
- **indexer**: Requires the signer to be either registered on the Graphcast Registry or to be a Graph Account, and corresponds to an Indexer satisfying the minimum stake requirement.
```
